DigitalOffice/Tests/TimeManagementServiceTests/Repositories/LeaveTimeRepositoryTests.cs

```diff
@@ -0,0 +1,94 @@
+using LT.DigitalOffice.TimeManagementService.Database;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Repositories;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Enums;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Repositories
+{
+    public class CreateLeaveTimeTests
+    {
+        private TimeManagementDbContext dbContext;
+        private ILeaveTimeRepository repository;
+
+        private Guid worker1;
+        private Guid worker2;
+
+        private DbLeaveTime testLeaveTime1;
+        private DbLeaveTime testLeaveTime2;
```

Comment:
We not separate fields (convention)

Reply:
this is not so important. These are unit tests

Reply:
Is it okay?
```
private TimeManagementDbContext dbContext;
private ILeaveTimeRepository repository;

private Guid worker1;
private Guid worker2;
private DbLeaveTime testLeaveTime1;
private DbLeaveTime testLeaveTime2;
```
Because the data for the test is not part of the tested functions.

Reply:
Fixed?